packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -532,7 +532,7 @@ export class DashboardService extends BaseService {
                 {
                     tiles: dashboard.tiles,
                     filters: dashboard.filters,
-                    tabs: dashboard.tabs,
+                    tabs: dashboard.tabs || [],
```

Comment:
not mandatory for this fix, as it is also patched in the frontend, but good to keep